web/themes/custom/novel/templates/layout/eventinstance--card.html.twig

```diff
@@ -1,12 +1,14 @@
-{% include '@novel/components/card.html.twig'
-  with {
-  attributes: attributes,
-  media: content.event_teaser_image,
-  variant: 'x-large',
-  has_media: (content.event_teaser_image.0),
-  teaser_text: content.event_teaser_text,
-  type_tag: 'Event'|trans,
-  date_tag: content.date,
-  title: content.title,
-  url: url('entity.eventinstance.canonical', {'eventinstance': eventinstance.id()})
-} only %}
+{% if not expired %}
```

Comment:
I understand your point but I do not think this is a good solution to the problem to place this logic in the template layer.

You suggest this solution to avoid having to implement the logic in every view but now we have to remember to put it in every template. I consider this is just as hard to maintain.

It may be easier to implement but on the other hand it messes with other parts of views dealing with the number of items in the list.

A view which is configured to show 5 events may now only show 4. A view with a pager with 10 events per page may now only show 8 per page even though it says that it is showing 10.

Reply:
Great point here, and I'm glad you called me out on it.
We've talked about this in person, and I've talked about it with Stefan.

This update (commit) has been reverted.

We came to the conclusion that we'll instead rely on automatic unpublication for events to not show up in various lists.

However, this will become a lot of changes, which I'll add in a seperate, new pull request.